packages/next/src/next-devtools/dev-overlay/components/devtools-panel/devtools-panel-tab/issues-tab/issues-tab-sidebar-frame-skeleton.tsx

```diff
@@ -0,0 +1,68 @@
+import { css } from '../../../../utils/css'
+
+export function IssuesTabSidebarFrameSkeleton({
+  isActive,
+  ...props
+}: {
+  isActive: boolean
+} & React.ButtonHTMLAttributes<HTMLButtonElement>) {
+  return (
+    <button
+      data-nextjs-devtools-panel-tab-issues-sidebar-frame
+      data-nextjs-devtools-panel-tab-issues-sidebar-frame-skeleton
+      data-nextjs-devtools-panel-tab-issues-sidebar-frame-active={isActive}
+      {...props}
+    >
+      <div
+        data-nextjs-devtools-panel-tab-issues-sidebar-frame-skeleton-bar
```

Comment:
you can use `data-nextjs-devtools-panel-tab-issues-sidebar-frame-skeleton-bar="<number>"` to distinguish different bar